scripts/process_twitter_data.py

```
@@ -0,0 +1,260 @@
+                return data
+            else:
+                print(f"⚠️  Could not extract JSON from {file_path.name}")
+                return None
+        else:
+            print(f"⚠️  File {file_path.name} doesn't appear to be a Twitter export")
+            return None
+            
+    except json.JSONDecodeError as e:
+        print(f"❌ JSON decode error in {file_path.name}: {e}")
+        return None
+    except Exception as e:
```

Ruff (0.14.1)

Do not catch blind exception: `Exception`

(BLE001)